src/components/GameCard.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react";
+import { Modal } from './Modal'
+
+export function GameCard({card,collection}){
+  const [modal,setModal] = useState(false)
+  return(
+    <div className = "gameCard">
+      <img {...card.image}
```

Comment:
better to provide property properly, rather than spreading it makes it more readable